File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        i, j = 0, len(numbers) - 1
        while i<j:
            if (numbers[i] + numbers[j]) > target:
                j = j-1
            elif (numbers[i] + numbers[j]) < target:
                i = i+1
            else: 
                return [i+1,j+1] 

File: Data Structures & Algorithms/binary-search/submission-1.java
class Solution {
    public int search(int[] nums, int target) {
        int front = 0, rear = nums.length - 1, mid;
        while (front <= rear){
            mid = (front + rear) /2;
            if (nums[mid] == target)
                return mid;
            else if (nums[mid] > target)
                rear = mid - 1;
            else
                front = mid + 1;
        }

        return -1;

    }
}
